azure-pipelines.yml

```diff
@@ -28,19 +38,31 @@ steps:
     arguments: '--configuration $(BuildConfiguration) --collect "Code coverage" /p:Threshold=58 /p:ThresholdStat=total /p:thresholdType=line'
   displayName: 'Running tests'
 
-# Create the deploy package
+# Create the deploy package for API
 - task: DotNetCoreCLI@2
   inputs:
     command: publish
-    publishWebProjects: True
-    arguments: '--configuration $(BuildConfiguration) --output $(Build.ArtifactStagingDirectory)'
-    zipAfterPublish: True
-  displayName: 'Build deployment package'
+    publishWebProjects: false
+    projects: '$(apiProject)'
+    arguments: '--configuration $(BuildConfiguration) --output $(System.DefaultWorkingDirectory)/Deploy/API'
+    zipAfterPublish: true
+    modifyOutputPath: true
+  displayName: 'Build API deployment package'
 
-# this code takes all the files in $(Build.ArtifactStagingDirectory) and uploads them as an artifact of your build.
-- task: PublishBuildArtifacts@1
+#Create the deploy package for HangFire
+- task: DotNetCoreCLI@2
   inputs:
-    pathtoPublish: '$(Build.ArtifactStagingDirectory)'
-    artifactName: 'DIGNDB.App.SmitteStop.API'
-  displayName: 'Publish build artifacts'
-
+    command: publish
+    publishWebProjects: false
+    projects: '$(hangFireProject)'
+    arguments: '--configuration $(BuildConfiguration) --output $(System.DefaultWorkingDirectory)/Deploy/HangFire'
+    zipAfterPublish: true
+    modifyOutputPath: true
+  displayName: 'Build HangFire deployment package'
+  
+# this uploads API as an artifact
```

Comment:
Could you start this comment with Uppercase letter i.e. `This` so it's consistent with other comments?